ADR/ADR023-use-uid-in-urls.md

```diff
@@ -0,0 +1,37 @@
+# ADR23: Use UIDs in URLs to replace form-id
+
+Date: 08/01/24
+
+## Status
+Proposed
+
+## Context
+
+See this spike for details about the required changes https://trello.com/c/BruuEN5Z/1225-spike-change-ids-in-urls-of-forms-to-not-be-the-primary-key
+
+We need to switch away from putting database ids in our URLs. Currently, form urls look something like this:
+`https://submit.forms.service.gov.uk/form/1234/apply-for-an-apple/5678`
+
+Where `1234` is the form-id.
+
+We instead want to change our URLs so that they look either like this: 
```

Comment:
Don't need this now that we've removed the other option:

```suggestion
We instead want to change our URLs so that they look like this: 
```

Reply:
good spot!